### PR TITLE
Fix index.html

### PR DESCRIPTION
**Pull request by JerryYu0815**

### CODE REVIEW

**Comment by JerryYu0815** [comment]
Should be no problems.

File: techeria_controller/templates/about.html [hunk 1]
```diff
@@ -3,171 +3,173 @@
...
-      © 2020 All Rights Reserved By
-      <a href="https://html.design/">Free Html Templates</a>
-    </p>
-  </section>
-  <!-- footer section -->
+    <!-- footer section -->
+    <section class="container-fluid footer_section">
+        <p>
+            © 2020 All Rights Reserved By
+            <a href="https://html.design/">Free Html Templates</a>
```
Comment (farahh001) [inline techeria_controller/templates/about.html:166]: we can get rid of this line

File: techeria_controller/templates/about.html [hunk 2]
```diff
@@ -3,171 +3,173 @@
...
-              <p>
-                demo@gmail.com
-              </p>
+    <!-- info section -->
+    <section class="info_section layout_padding">
+        <div class="container">
+            <div class="info_items">
+                <a href=" {% static '' %} ">
+                    <div class="item ">
+                        <div class="img-box box-1">
```
Comment (farahh001) [inline techeria_controller/templates/about.html:120]: is this static taking any images?

File: techeria_controller/templates/about.html [hunk 3]
```diff
@@ -3,171 +3,173 @@
...
+    <div class="bg">
+
+        <!-- about section -->
+        <section class="about_section layout_padding">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-md-5 offset-md-2">
+                        <div class="about_detail-box">
+                            <h3 class="custom_heading ">
+                                As an e-commerse website, techeria...
```
Comment (farahh001) [inline techeria_controller/templates/about.html:81]: E-commerce website! spelling mistake

File: techeria_controller/templates/about.html (hunk 1 above)
Comment (JerryYu0815) [reply techeria_controller/templates/about.html:166]: Sure

File: techeria_controller/templates/about.html (hunk 2 above)
Comment (JerryYu0815) [reply techeria_controller/templates/about.html:120]: The image is configued in .css file.

**Pull request merged**